test/unit/lib/migration-chunks/validation/field/change-id.spec.js

```diff
@@ -258,4 +258,43 @@ describe('chunks validation when changing field ids', () => {
       }
     ]);
   }));
+
+  it('when changing the ID of a field but that field does not exist', Bluebird.coroutine(function * () {
```

Comment:
maybe smth like `it('fails when changing the ID of a field that does not exist')`, to preserve the "fluent readability"